### PR TITLE
chore: session replay for rn

### PR DESCRIPTION
**Pull request by marandaneto**
## Problem

Closes https://github.com/PostHog/posthog/issues/13269

### Bump level

- [ ] Major
- [X] Minor
- [ ] Patch

### Libraries affected

- [ ] All of them
- [ ] posthog-web
- [ ] posthog-node
- [X] posthog-react-native

### Changelog notes

- Added support for X

### CODE REVIEW

**Review by pauldambra — APPROVED** [review]
excite!

**Review by daibhin — APPROVED** [review]
🥳

File: posthog-react-native/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
 1. chore: default `captureMode` changed to `json`.
     1. To keep using the `form` mode, just set the `captureMode` option to `form` when initializing the PostHog client.
+2. chore: Session Replay for React-Native - Experimental support
+
+Install Session Replay for React-Native:
+
+```bash
+yarn add posthog-react-native-session-replay
+# or yarn add
+npm i -s posthog-react-native-session-replay
```
Comment (marandaneto) [inline posthog-react-native/CHANGELOG.md:16]:
```suggestion
# or npm
npm i -s posthog-react-native-session-replay
```

**Comment by wowitsjerome** [comment]
is this real life?

**Pull request merged**